README.md

```diff
@@ -1 +1,192 @@
+- **[Search Feature]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- Link: [Live Demo](https://dota2-heros.netlify.app/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A Chrome browser
+- npm
+- Node.js
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/K0ppai/dota2_heros.git
+```
+
+### Install
+
+Install the prerequisites this project using:
+
+```sh
+  cd dota2_heros
+
+  npm install --save-dev eslint@7.x eslint-config-airbnb@18.x eslint-plugin-import@2.x eslint-plugin-jsx-a11y@6.x eslint-plugin-react@7.x eslint-plugin-react-hooks@4.x @babel/eslint-parser@7.x @babel/core@7.x  @babel/plugin-syntax-jsx@7.x  @babel/preset-react@7.x @babel/preset-react@7.x
+
+  npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```
+  npm run start
+```
+
+### Run tests
+
+To run tests after setting up the linters, run the following command:
+
+```sh
+  npx eslint "**/*.{js,jsx}" --fix
+```
+```sh
+  npx stylelint "**/*.{css,scss}" --fix
+```
+For tests, run:
+
+```sh
+  npm run test
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Paing Soe Thu**
+
+- GitHub: [@K0ppai](https://github.com/K0ppai)
+- Twitter: [@_koppai_](https://twitter.com/_koppai_)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/paingsoe-thu)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[Mobile Responsiveness]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project, don't forget to give it a ⭐️!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+
+I would like to give credit to the original designer of the layout [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) and [dota2.com](https://www.dota2.com/home) for the images and design.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```

Comment:
- [ ] Please kindly fix this broken link for your README file to be engaging and professional. 👍 **Perhaps you need to create a LICENSE file or you link it wit MIT.md**